hooks/terraform_wrapper_module_for_each.sh

```diff
@@ -321,14 +321,14 @@ EOF
 
     # Get names of module variables in all terraform files
     # shellcheck disable=SC2207
-    module_vars=($(echo "$all_tf_content" | hcledit block list | { grep variable. | cut -d'.' -f 2 | sort || true; }))
+    module_vars=($(echo "$all_tf_content" | hcledit block list | { grep "^\s*variable." | cut -d'.' -f 2 | sort || true; }))
```

Comment:
```suggestion
    module_vars=($(echo "$all_tf_content" | hcledit block list | { grep "^variable." | cut -d'.' -f 2 | sort || true; }))
```
This should be enough. I don't think there can/should be any space before the "variable".

Please apply the same fix on other lines in this PR.

Reply:
> I don't think there can/should be any space before the "variable".

Leading spaces do not produce errors, trigger failures and are not otherwise disallowed explicitly. Hence should be considered allowed imho.
On the other end `hcledit block list` seems to strip leading spaces, which means Anton's recommendation is correct =)

Reply:
I also would like to recommend to escape dot to not mix it with "any char" in this use case:
```suggestion
    module_vars=($(echo "$all_tf_content" | hcledit block list | { grep "^variable\." | cut -d'.' -f 2 | sort || true; }))
```

Reply:
And as Anton mentioned: please apply the same to other code bits in this PR.

Reply:
Done! Please resolve if this looks sufficient. Thanks!